src/main/resources/stashpullrequestbuilder/stashpullrequestbuilder/StashBuildTrigger/help-stashHost.html

```diff
@@ -0,0 +1,6 @@
+<div>
+    The http or https URL of the Stash server REST API.
```

Comment:
Should here be a trailing `<br/>` before the NOTE?

Reply:
Done